languages/common-lisp/exercises/concept/sexpr/sexpr.lisp

```diff
@@ -0,0 +1,17 @@
+(defpackage #:sexpr
+  (:use :cl)
+  (:export :is-an-atom-p :is-a-cons-p :first-thing :rest-of-it))
+
+(in-package :sexpr)
+
+(defun is-an-atom-p (thing)
+  "Evaluates to T if THING is an atom, NIL otherwise")
+
+(defun is-a-cons-p (thing)
+  "Evaluates to T if THING is a cons, NIL otherwise")
+
+(defun first-thing (cons)
```

Comment:
Common lisp is a Lisp-2, so it's totally fine to name this variable "cons" but it's possible that could be confusing?

Would it be worth renaming it to something like `cons-cell` or something else?

Reply:
well `cons-cells` would be more correct as it has more than one cons cell in it (a cons always has 2 cells).

I don't feel too strongly about it. I guess I had half of an idea that we could *show* rather than teach at this point about Common Lisp being a Lisp2.

Reply:
Yeah, I actually don't mind that too much. Let's leave it as is for now?

We can always change it later if it confuses people, but I think that it should be fine.